rewrite-maven/src/test/java/org/openrewrite/maven/ChangeParentPomTest.java

```diff
@@ -1450,22 +1451,13 @@ void doesNotBringInMavenArguments() {
                     </parent>
                   </project>
                   """,
-                """
-                  <project>
-                    <groupId>org.sample</groupId>
-                    <artifactId>sample</artifactId>
-                    <version>1.0.0</version>
-                    <properties>
-                      <maven.repo.local>${user.home}/.m2/repository</maven.repo.local>
-                      <owaspDb>${maven.repo.local}/blah</owaspDb>
-                    </properties>
-                    <parent>
-                      <groupId>org.springframework.boot</groupId>
-                      <artifactId>spring-boot-starter-parent</artifactId>
-                      <version>3.5.9</version>
-                    </parent>
-                  </project>
-                  """
+                spec -> spec.after(actual -> {
+                    assertThat(actual)
+                      .as("Any version of Spring boot parent above `3.5.7`")
+                      .doesNotContain("3.5.7")
```

Comment:
The intention of this test wasn't the dependency upgrade, but to not swap in the actual property value. We should add assertions that these are maintained: 
```
<maven.repo.local>${user.home}/.m2/repository</maven.repo.local>
<owaspDb>${maven.repo.local}/blah</owaspDb>
```